modules/llm-cache/ds/kv_state_cache_block.h

```diff
@@ -29,11 +29,26 @@ limitations under the License.
 #include "client/ds/i_object.h"
 #include "llm-cache/radix-tree/radix-tree.h"
 
+// Set the bit to 1, which means the resource is not being used
+#define FREE_BIT_RESOURCE(value, bit) ((value) |= (((uint64_t) 1) << (bit)))
+
+// Set the bit to 0, which means the resource is being used
+#define ACQUIRE_BIT_RESOURCE(value, bit) \
+  ((value) &= (~(((uint64_t) 1) << (bit))))
+
+#define DEFAULT_BLOCK_SIZE 64
```

Comment:
Use a `constexpr`.

Reply:
OK.
Does “ACQUIRE_BIT_RESOURCE” need to be a constexpr function as well?

Reply:
I'm afraid it cannot be a constexpr function, as it argument `value` cannot be a constant.